### PR TITLE
docs: bring in main site content to dev site

### PR DESCRIPTION
**Pull request by lazd**
## Description

This PR brings the dev site up to snuff with the content on the existing public site.

## Screenshots

image

image

## To-do list

- [x] If my change impacts other components, I have tested to make sure they don't break.
- [x] If my change impacts documentation, I have updated the documentation accordingly.
- [x] I have read the [CONTRIBUTING document](/.github/CONTRIBUTING.md).

- [x] Fixup layout differences
- [x] Add Adobe footer
- [x] Find a way to run JS on FastLoad
- ~[ ] See if there is a less disgusting way to do code samples in Get Started page~
- [x] Design review by Spectrum Team
  - [x] Give screenshot a min/max height on Get started page
  - [x] Dark theme for syntax highlighted colors
- [x] This pull request is ready to merge.

### CODE REVIEW

File: site/get-started.pug [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/vars/dist/spectrum-light.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/page/dist/index-vars.css">
+                    <link rel="stylesheet" href="node_modules/@spectrum-css/button/dist/index-vars.css">
+                  </head>
+                  <body>
+                    <button class="spectrum-Button spectrum-Button--cta">
+                      <span class="spectrum-Button-label">Button</span>
+                    </button>
+                  </body>
+                </html>`, util.Prism.languages.markup, 'markup');
```
Comment (lazd) [inline site/get-started.pug]: Ok I know this is pretty gross, but its the best way I could come up with. We should definitely change it later.

File: site/includes/nav.pug [hunk 2]
```diff
@@ -1,8 +1,8 @@
 .u-scrollable.spectrum-Site-nav
   nav
     ul.spectrum-SideNav.spectrum-SideNav--multiLevel
-      //- li.spectrum-SideNav-item(class=pageURL === 'getting-started.html' ? 'is-selected' : '')
-      //-   a.spectrum-SideNav-itemLink.js-fastLoad(href='getting-started.html') Getting Started
+      li.spectrum-SideNav-item(class=pageURL === 'getting-started.html' ? 'is-selected' : '')
+        a.spectrum-SideNav-itemLink.js-fastLoad(href='get-started.html') Get Started
```
Comment (lazd) [inline site/includes/nav.pug]: Note this is a URL change -- it's simply `/get-started` now

File: site/index.pug [hunk 3]
```diff
@@ -16,12 +16,40 @@ html(lang='en-US' dir="ltr").spectrum.spectrum--light.spectrum--medium
...
+                img(src="img/illustration_tested.svg" alt="Tested illustation")
+                div
+                  h4.spectrum-Heading.spectrum-Heading--S Rigorously tested
+                  p.spectrum-Body.spectrum-Body--M These individually-versioned components have been vetted to be accessible and inclusive of global audiences.
+
+              div.spectrum-HomeCard
+                img(src="img/illustration_flexible.svg" alt="Flexible illustation")
+                div
+                h4.spectrum-Heading.spectrum-Heading--S Multi-platform support
+                  p.spectrum-Body.spectrum-Body--M We support evergreen browsers (minus one version) and IE 11 for scalability and flexibility.
```
Comment (lazd) [inline site/index.pug]: These little illustrations don't have the responsive layout that the existing site does, that will need to be added.

**Comment by lazd** [comment]
@GarthDB any insight on the easiest way to add the Adobe footer here?

**Comment by jianliao** [comment]
@lazd Did you resolve the adobe footer issue? If not, I can help.

**Comment by lazd** [comment]
@jianliao I did not, go ahead and push commits directly to this branch! Thank you!

**Comment by jianliao** [comment]
@lazd Please check latest dev site [build](https://opensource.adobe.com/spectrum-css-vr-results/8b85bc6-result-2020-05-18_21-41-55/dist/docs/) to check the footer. There are some [options](site/includes/adobe-globalnav.js) to customize the footer. Let me know if there is anything you want to change.

```
theme:{
    backgroundColor:'#FFFFFF',
    columnTitleColor: '#00FF00',
    linkColor: '#4B4B4B',
    linkHoverColor: '#2C2C2C',
    separatorColor: '',
    caretColor: ''
}
```
But you have to deploy the site to *.adobe.com to see the change.

File: components/site/index.css [hunk 4]
```diff
@@ -184,14 +184,8 @@ governing permissions and limitations under the License.
...
-.spectrum-Site-footer {
-  display: flex;
-  flex-direction: row;
-  justify-content: space-between;
-}
-.spectrum-Site-footerContent {
-}
-.spectrum-Site-footerCopyright {
+.spectrum-Site-footerContainer {
+  /* TODO place footer appear under the .spectrum-Site-mainContainer */
```
Comment (jianliao) [inline components/site/index.css:188]: @lazd What should I put it here to display the footer under the `spectrum-Site-mainContainer`?

**Comment by jianliao** [comment]
@lazd I found that we need to move global footer out of the `spectrum-Site-mainContainer` as the footer script does not work well with fast-load. So I created a div with class `spectrum-Site-footerContainer` to wrapper the entire footer and move it out of the mainContainer. But I could not figure out how to make `spectrum-Site-footerContainer` display under the mainContainer. Please see my inline comment on `components/site/index.css`

**Comment by lazd** [comment]
Hmm @jianliao seeing the footer appear on the right side of the screen on the latest devsite docs
image

**Pull request merged**